01-filedir.md

```diff
@@ -342,13 +333,21 @@ Desktop/            north-pacific-gyre/ writing/
 
 `-a` stands for "show all";
 it forces `ls` to show us file and directory names that begin with `.`,
```

Comment:
Add a half-sentence that files and folders starting with '.' are hidden by default (in shell and Finder)?